Reconstruction/RecCalorimeter/tests/options/runFullCaloSystem_SimAndFullDigitisation.py

```diff
@@ -0,0 +1,197 @@
+import os
+import numpy as np
+
+# simulations setup                                                                                                                                                                                                              
+energy=1000000
```

Comment:
Why do you run for such high energy? Is it that you want to have signal in most of the detector in one direction? Maybe the energy could be lowered for the testing.

Reply:
I kept it high to make sure I have deposits in all sub detectors that I have tested.. but I can lower it now.